contracts/jetton/master.tact

```diff
@@ -52,7 +53,14 @@ contract JettonMaster with TEP74JettonMaster, TEP89JettonDiscoverable, Deployabl
         
         // Update max supply
         if (msg.key == "max_supply") {
-            self.max_supply = msg.value.loadCoins();
+            //Cannot set max supply to 0 through this method
+            let new_max_supply = msg.value.loadCoins();
+            nativeThrowIf(ERROR_CODE_INVALID_AMOUNT, new_max_supply <= 0);
+
+            // Ensure the new max supply is greater than the current max supply
+            nativeThrowIf(ERROR_CODE_INVALID_AMOUNT, new_max_supply <= self.max_supply);
```

Comment:
Does first check is required? Could we combine it in one `nativeThrowIf`?

Because of `max_supply` is 0 so new value cannot be lower than 0 because of second check.
Also if `max_supply` is any other value, so it can't be set to 0 because 0 is lower than current value

```suggestion
            nativeThrowIf(ERROR_CODE_INVALID_AMOUNT, new_max_supply <= self.max_supply);
```

Reply:
Initial thought was to use 2 different exit codes for the 2 cases, but i ended up using 1 so lets get rid of the extra 😃